unit-tests/Extension/ExitDieTest.php

```diff
@@ -23,6 +23,8 @@ public function testExitDie()
             $phpBinary .= ' -qrr';
         }
 
+        $phpBinary .= " -d 'enable_dl=true'";
+
```

Comment:
@danhunsaker This deprecated feature will certainly be removed in the future: http://php.net/manual/en/info.configuration.php Why do we need this?